src/Components/Timer.tsx

```diff
@@ -10,13 +10,13 @@ const Timer = () => {
   console.log('Render timer') // 28 times
   const { gameState, gameDispatch } = useGame();
   const gameOver = useGameOverDocState();
-  const gamePaused = useGamePausedDocState();
-  const { weaponsStolen } = useWeaponsStolenDocState();
+  const sandTimerCount = useSandTimerState();
+  const weaponsStolen = useWeaponsStolenDocState();
   const time = new Date();
   const { gameAudio, loadEscapeSoundtrack, setGameAudio, playWarningSound, musicOn, loadGameSoundtrack } = useAudio();
 
   useEffect(() => {
-    if (weaponsStolen) {
+    if (weaponsStolen.length === 4) {
```

Comment:
I've made this change on another pr that's already merged to main. weaponsStolen should be a boolean